Core/HLE/sceAtrac.cpp

```diff
@@ -577,9 +577,9 @@ int Atrac::Analyze() {
 	int loopFirstSampleOffset = 0;
 	firstSampleoffset = 0;
 
-	while (Memory::Read_U32(first.addr + offset) != RIFF_WAVE_MAGIC) {
+	while (Memory::ReadUnchecked_U32(first.addr + offset) != RIFF_WAVE_MAGIC) {
```

Comment:
Oops, yeah, these changes weren't meant to go in this pull request. I'll remove them.